### ISSUE TITLE
Allow for more than one type of media download for QR code component

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#473]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

After #470 is merged the goal will be to add the ability to download the QR code from that links to an organization or event page. This work is covered by #472 :) This issue will be to finalize the QR code component by adding in a second button with dropdown options that will allow other filetypes to be downloaded. The default file type for clicking `Download QR code` will be switched to a `.png`, and the dropdown will allow for downloading `.png` (so they see it's an option), `.svg` and `.jpeg`.

- Please note that this issue is blocked by #353 and #472

A general image of how the CTA button for the component will look after this issue is:

![Screenshot_2023-10-18 00 59 21](https://github.com/activist-org/activist/assets/24387426/607c1a99-1cf9-4dde-9999-311a4b8f0d06)

### Contribution

Potentially @VinnyHS2 has interest in this one as well :) Happy to support someone who wants to work on this!

**Comment by mjiruobe** [comment]
I'll try to solve this issue :-)

**Comment by andrewtavis** [comment]
Thanks @mjiruobe! Let us know if we can do anything to support 😊😊

**Comment by andrewtavis** [comment]
Hey @mjiruobe 👋 Quick note on this that I just realized: the QR code modal is currently not reachable via tabs. Could you also try to fix that? I can also make another issue once this is done if you'd prefer :) Accessibility wise it's good once we open it in so far as the close button is focused and we can close it with escape as well, but we just need to be able to get there. Loads of work to be done on this topic, but regardless would be good to get this one done 😊

**Comment by andrewtavis** [comment]
Closed via #565 with some final work done in 6e637a9. Thanks for all the work here, @mjiruobe! 🚀
